agent/fw_wordpress.c

```diff
@@ -578,6 +578,21 @@ NR_PHP_WRAPPER(nr_wordpress_apply_filters) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_wordpress_version(TSRMLS_D) {
+  char* string = "$GLOBALS['wp_version'];";
+  zval retval;
+  int result = zend_eval_string(string, &retval,
+                                "Retrieve Wordpress Version" TSRMLS_CC);
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      // Add php package to transaction
+      nr_txn_add_php_package(NRPRG(txn), "wordpress", version);
+      zval_dtor(&retval);
+    }
+  }
```

Comment:
Should the agent still report wordpress as a detected package w/o a version if the version check fails?

Reply:
Yes, thanks for catching that. Fixed: [3eaa90f](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/3eaa90f72f9976ab895574749ae180d1acad0c0a)